rep-0145.rst

```diff
@@ -65,6 +65,8 @@ Data Sources
   - The accelerometers report linear acceleration data expressed in the sensor frame of the device. This data is output from the driver as a 3D vector, representing the specific force acting on the sensor.
 
   - When the device is at rest, the vector will represent the specific force solely due to gravity. I.e. if the body `z` axis points upwards, its `z` axis should indicate +g. This data must be in m/s^2.
+  
+  - Specific force is the non-gravity-induced acceleration of a body (or acceleration "relative to" free fall). That is why the specific force of a body in rest is +g (the specific force must be the opposite of gravity to keep the body in rest). Specific force of a free falling body is zero.
```

Comment:
I've added this paragraph mostly as a quick explainer distilled from Wikipedia for readers who do not know much about specific forces. So it is not meant as a specification. You are right that there is overlap with the previous paragraph.

What about this condensation of the 3 paragraphs into 2?

  - The accelerometers report linear acceleration data expressed in the sensor frame of the device. This data is output from the driver as a 3D vector, representing the specific force acting on the sensor. This data must be in m/s^2.

  - [Definition] Specific force is the non-gravity-induced acceleration of a body (or acceleration "relative to" free fall). That is why the specific force of a body in rest is +g (the specific force must be the opposite of gravity to keep the body in rest). Specific force of a free falling body is zero.

Reply:
Apologies, this is on a hiatus for me given the recent changes in employment. Need to baton down the hatches a bit and revisit things like this once I know what's going on

Reply:
I understand that, Steve. Would you be willing to merge this PR as is to https://github.com/ros-infrastructure/rep/pull/372, so that the discussion is not so fragmented? Alternatively, I could open a new PR for the REP and transfer all the activity there and drag the cart for a while instead of you ;) What do you prefer?